src/cvar.h

```diff
@@ -49,6 +49,7 @@ _Bool Cvar_Command(void);
 char *Cvar_UserInfo(void);
 char *Cvar_ServerInfo(void);
 void Cvar_WriteAll(file_t *f);
+_Bool Cvar_EmplaceValues(const char *input, const size_t in_size, GString **output);
```

Comment:
If you're not thrilled with this name, I'd suggest Cvar_ExpandString. But up to you.